fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/SkolemNodeRdfContext.java

```diff
@@ -93,6 +94,10 @@ public SkolemNodeRdfContext(final FedoraResource resource,
             return null;
 
         } catch (final RepositoryException e) {
+            if (e instanceof ItemNotFoundException) {
+                // Effectively skip resources that do not exist
```

Comment:
Thinking about this a bit more, I can see the desire for code that compensates for data "corrupted" by this bug - that would allow this fix to be dropped over an existing repository and Just Work.  However, would it be then be necessary (or worth it) to add some tests to verify that the desired compensation works as intended?